pantheon-bundle/src/test/java/com/redhat/pantheon/servlet/ModuleJsonServletTest.java

```diff
@@ -60,7 +61,7 @@ void getQuery() {
 
 
     @Test
-    void resourceToMap() {
+    void resourceToMap() throws RepositoryException {
```

Comment:
Just change it to `throws Exception`.
Tests should not throw any exception at all, and this would prevent you from adding more Exception types to the method definition in the future.

Reply:
updated